packages/types/src/userSettings.ts

```diff
@@ -101,7 +102,6 @@ export interface UserSettingsJSON extends ClerkResourceJSON {
   sign_in: SignInData;
   sign_up: SignUpData;
   password_settings: PasswordSettingsData;
-  actions: Actions;
```

Comment:
This is just a move to make the placement consistent between the resource and json types